package.json

```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node index.js",
+    "dev": "nodemon index.js"
```

Comment:
This should have been added with the code base not in a branch.
(and the code base didn't need to be added through a branch so my delayed review is not the problem here.🙂)